File: src/main/scala/tk/fishfish/dataflow/Bootstrap.scala
package tk.fishfish.dataflow

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.scheduling.annotation.EnableAsync
import tk.fishfish.mybatis.enums.EnableEnumTypes

/**
 * 启动
 *
 * @author 奔波儿灞
 * @version 1.0.0
 */
@EnableAsync
@EnableEnumTypes
@SpringBootApplication
class Bootstrap {
}

object Bootstrap {

  def main(args: Array[String]): Unit = {
    try {
      SpringApplication.run(classOf[Bootstrap], args: _*)
    } catch {
      case e: Throwable => {
        println("启动异常", e)
      }
    }
  }

}

File: src/main/java/tk/fishfish/dataflow/Bootstrap.java
package tk.fishfish.dataflow;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import tk.fishfish.mybatis.enums.EnableEnumTypes;

/**
 * 入口
 *
 * @author 奔波儿灞
 * @version 1.0.0
 */
@EnableEnumTypes
@SpringBootApplication
public class Bootstrap {

    public static void main(String[] args) {
        SpringApplication.run(Bootstrap.class, args);
    }

}
